Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/Magazines/Rifles.yml

```diff
@@ -0,0 +1,74 @@
+- type: entity #made By VividPups added in with PR 1770
+  id: MagazineSVT
+  name: "10 rounder SVT magazine (.30 rifle)"
```

Comment:
Name strikes me as odd.

If it's only for the SVT, rename to "SVT magazine (.30 rifle)" and fix the tag here (e.g. SAW mag being MagazineLightRifleBox vs MagazineLightRifle).
If not (currently interchangeable with the AKMS), rename to "10 round magazine (.30 rifle)".

Also, consider that even the SAW magazine's ID doesn't name the SAW at all.